jshift-kit-vertx/src/main/java/io/jshift/vertx/enricher/VertxHealthCheckEnricher.java

```diff
@@ -358,4 +358,16 @@ private Probe discoverVertxHealthCheck(boolean readiness) {
         return Optional.of(root);
     }
 
-}
\ No newline at end of file
+    private Optional<Map<String, Object>> getMavenPluginConfiguration() {
+        final String[] jshiftPlugins = {"io.jshift:kubernetes-maven-plugin", "io.jshift:openshift-maven-plugin"};
```

Comment:
In the comment I mean to so it as a constant on class level with `static` and `final` modifiers.